src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -138,6 +158,25 @@ public function send(string $body, array $headers, int $delay = 0): string
 
     public function get(): ?array
     {
+        if ($this->usePgsqlNotify && null !== $this->queueEmptiedAt) {
```

Comment:
this needs to go inside the try block below, and work nicely with autosetup. the table might not exist yet.

Reply:
I'm not sure to follow.

Reply:
the table specified in`$this->configuration['table_name']` might not be created yet.
If `auto_setup` is enabled, and the table has not yet been created, it'll be done on line 225.

Reply:
It doesn't matter. The channel name can be any random string. It's a common convention (documented in Postgres) to reuse the table name as channel name (as done here), but it's not a hard requirement, and if the table not exist, `LISTEN` will work anyway. (anyway, here we're sure that the table will exist before the call to `LISTEN`)

Reply:
gotcha, thanks!